yang/frr-zebra.yang

```diff
@@ -2367,6 +2367,18 @@ module frr-zebra {
             "RFC 4861: Neighbor Discovery for IP version 6 (IPv6)
                        - AdvReachableTime";
         }
+        leaf retrans-timer {
+          type uint32;
+          units "milliseconds";
+          default "0";
+          description
+            "The value to be placed in the Retrans Timer field in
+            the Router Advertisement messages sent by the router.
+            A value of zero means unspecified (by this router).";
```

Comment:
Same comment as earlier, why not just use lack of presence for "dont' advertise"?

Reply:
This case is different from MTU. `Retrans Timer` is not an option, it is a mandatory field that can be set to zero.